Watchman.Discord/Areas/Users/Controllers/UsersController.cs

```diff
@@ -11,6 +11,7 @@
 using Watchman.Discord.Areas.Users.Services;
 using Devscord.DiscordFramework.Commons.Exceptions;
 using Watchman.Discord.Areas.Users.BotCommands;
+using Watchman.DomainModel.Users.Commands.Handlers;
```

Comment:
github źle mi pokazuje, czy to jedyna zmiana w tym pliku?

Reply:
jedyna, nie wiem jak to zrobiłem

Reply:
czemu dałeś resolved skoro nie poprawiłeś?

Reply:
może błędnie, ale założyłem że to było takie tylko wspomnienie, a że "zwykły import" jest mało istotny, teraz poprawiłem